.gitignore

```diff
@@ -1 +1,2 @@
-node_modules
\ No newline at end of file
+node_modules
+build
```

Comment:
We want to commit build files in this instance

Reply:
Taking care in the next commit.